cmd/lakectl/cmd/ingest.go

```diff
@@ -81,5 +135,6 @@ func init() {
 	_ = ingestCmd.MarkFlagRequired("to")
 	ingestCmd.Flags().Bool("dry-run", false, "only print the paths to be ingested")
 	ingestCmd.Flags().BoolP("verbose", "v", false, "print stats for each individual object staged")
+	ingestCmd.Flags().IntP("concurrency", "C", 64, "max concurrent API calls to make to the lakeFS server")
```

Comment:
Why capital 'C'?
```suggestion
	ingestCmd.Flags().IntP("concurrency", "c", 64, "max concurrent API calls to make to the lakeFS server")
```

Reply:
because we use `-c` on the global `lakectl` command as short for `--config`